TestingApp/ViewController/ViewController+CallVisualizer.swift

```diff
@@ -12,12 +12,12 @@ extension ViewController {
     }
     
     func showVisitorCode(presentation: CallVisualizer.Presentation) {
-        showRemoteConfigAlert { [weak self] fileName in
-            let config = self?.retrieveRemoteConfiguration(fileName)
+//        showRemoteConfigAlert { [weak self] fileName in
```

Comment:
I guess this can be removed

Reply:
fixed - 5e1bb4ca45e52a1fdceb2d4543687395dcaccf66